### PR TITLE
Remove macOS 11 logger

### PR DESCRIPTION
**Pull request by inderdhir**

### CODE REVIEW

File: DatWeatherDoe/API/WeatherRepository.swift [hunk 1]
```diff
@@ -61,9 +54,8 @@ final class WeatherRepository: WeatherRepositoryType {
 
         URLSession.shared.dataTask(with: url) { [weak self] data, _, error in
             guard let `self` = self, let data = data, error == nil else {
```
Comment (dlafreniere) [inline DatWeatherDoe/API/WeatherRepository.swift:56]: this results in a compilation error for me:
`Variable declared in 'guard' condition is not usable in its body`

using something like this works:

```
guard let strongSelf = self else {
    self?.logger...
  
    strongSelf.parseResponse...
}
```

File: DatWeatherDoe/API/WeatherRepository.swift [hunk 2]
```diff
@@ -126,9 +114,8 @@ final class WeatherRepository: WeatherRepositoryType {
...
```
Comment (dlafreniere) [inline DatWeatherDoe/API/WeatherRepository.swift:116]: ditto

**Review by dlafreniere — APPROVED** [review]
awesome, thanks!

Fixes https://github.com/inderdhir/DatWeatherDoe/issues/32

**Comment by inderdhir** [comment]
@dlafreniere Question: What Xcode version are you using? You may be using an older version with an older version of Swift which might explain why `guard let self = self` breaks.

**Comment by dlafreniere** [comment]
> @dlafreniere Question: What Xcode version are you using? You may be using an older version with an older version of Swift which might explain why `guard let self = self` breaks.

12.4 it's the latest version that works on Catalina

**Comment by inderdhir** [comment]
@dlafreniere Just pushed a change to this branch, let me know if that compiles for ya and I'll cut another release

**Comment by dlafreniere** [comment]
@inderdhir it's all good! 💯

**Pull request merged**